pkg/service/repair/plan.go

```diff
@@ -371,3 +376,28 @@ func newHostKsTable(host, ks, table string) scyllaclient.HostKeyspaceTable {
 		Table:    table,
 	}
 }
+
+func isSmallTableOptSupported(ctx context.Context, client *scyllaclient.Client, hosts []string) (bool, error) {
+	out := atomic.Bool{}
+	out.Store(true)
+
+	err := parallel.Run(len(hosts), parallel.NoLimit, func(i int) error {
+		h := hosts[i]
+		ni, err := client.NodeInfo(ctx, h)
+		if err != nil {
+			return parallel.Abort(err)
+		}
+		res, err := ni.SupportsRepairSmallTableOptimization()
+		if err != nil {
+			return parallel.Abort(err)
+		}
+		if !res {
+			out.Store(false)
+		}
+		return nil
+	}, parallel.NopNotify)
```

Comment:
It doesn't have much sens to use inhouse parallel.Run here IMHO
This https://pkg.go.dev/golang.org/x/sync/errgroup serves exactly the same purpose.